library/CHART-TEMPLATE/templates/deployment.yaml

```diff
@@ -23,15 +23,19 @@ spec:
         {{- . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "common.serviceAccount.name" . }}
+      {{- with include "common.securityContext.pod" . }}
       securityContext:
```

Comment:
Something thorny here is what will we do with the `privileged` flag, as it _needs_ to override some things in the security context (e.g. the `securityContext.privileged` flag). However I'm not sure if we want to discuss that in _this_ PR.

Reply:
We could talk about defaults changing if defaults "depending-on-things" in the `nri-kubernetes` chart.